### ISSUE TITLE
ERR when invitation

### ISSUE THREAD

**Issue by WhiteSeasons** [openwallet-foundation/bifold-wallet#68]
1、run Bob/Ailice agent from "aries-acapy-plugin-toolbox " project，it seems nothing wrong ：

![D9C4A6AC-046E-4b87-AD92-64EFD0E7AB8E](https://user-images.githubusercontent.com/87514220/128156967-fd0ca56d-769b-4360-b709-20a12c2cad71.png)

2、set the “event.data” param in the QRCodeScanner interface。

![E975CF7F-6C18-4d1b-A308-DCA965B0EE9C](https://user-images.githubusercontent.com/87514220/128157872-619070ef-e75c-4284-b002-f92a6cb2c5e4.png)

3、run the mobile agent ，error occured ：
![74BD40D3-768F-42ea-AADA-6B46617A2304](https://user-images.githubusercontent.com/87514220/128157995-5c67274f-9d42-445c-a9d2-0bb942ac93c5.png)

So what's the problem，and give some suggestion。
Thank you！

**Comment by phearaeun** [comment]
I guess `undefined` connections here happens when the agent is not initiated. I got the same error but during provisioning.
If the provisioning removed, the agent can be initiated but another issue coming :(.

**Comment by WhiteSeasons** [comment]
> I guess `undefined` connections here happens when the agent is not initiated. I got the same error but during provisioning.
> If the provisioning removed, the agent can be initiated but another issue coming :(.


yes,  agent is not initiated.  problems occured when running this code in agent init() interface：

        if (genesisPath) {
            await this.ledger.connect(poolName, {
                genesis_txn: genesisPath,
            });
        }
        if (this.inboundTransporter) {
            await this.inboundTransporter.start(this);
        }

**Comment by phearaeun** [comment]
In my case, I just removed provisioning part, agent can be initiated. Accepting new invite also works. But there is a problem with receiving messages from mediator. I guess the provision part needs to be done first before making new connection. I am still figuring out on this part too. My problem right now is the provision not work as it keeps showing error [here](https://github.com/hyperledger/aries-mobile-agent-react-native/issues/69)
